### PR TITLE
Understanding Dart Sass modules and namespaced variables

### PR DESCRIPTION
**Pull request by hougasian**

### CODE REVIEW

File: _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+twitter_text: "Understanding Dart Sass modules and name spaced variables"
+authors: Kevin Hougasian
+image: https://images.unsplash.com/photo-1496715976403-7e36dc43f17b?w=1000
+image_url: https://unsplash.com/photos/W8KTS-mhFUE
+image_credit: "@billy_huy"
+---
+
+For a long time, we were dependent on the CSS frameworks of others -- and we were quite happy. 😀
+
+As our needs grew, we needed a framework that suited all of them. So we created [Platform UI](https://style.rimdev.io), our custom framework written Sass, BEM, and vanilla JS for our static sites and application platform.
```
Comment (kendaleiv) [inline _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md]:
```suggestion
As our needs grew, we needed a framework that suited all of them. So we created [Platform UI](https://style.rimdev.io), our custom framework written using Sass, BEM, and vanilla JS for our static sites and application platform.
```

File: _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+Sass recently launched the new [Dart Sass module system](https://github.com/sass/dart-sass/releases/tag/1.23.0), which [Miriam Suzanne](@mirisuzanne) did a great job of introducing in a recent [CSS-Tricks post](https://css-tricks.com/introducing-sass-modules/).
```
Comment (kendaleiv) [inline _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md]: Do we have something to make `@mirisuzanne` a link? Or, can you change it to a Twitter link?

File: _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+...
+```
+
+## Summary
+
+The new Dart Sass module system has offered us better methods to accomplish the goals of our framework as it continues to grow. The port provides greater transparency into variables and _where_ we're calling other files.
+
+The conversion was relatively straight-forward and offered us the opportunity to review our current code base and shift core concepts while not altering the resulting compiled CSS.
+
+I encourage all to embrace Sass modules! Here are a few links to get you can get started:
```
Comment (kendaleiv) [inline _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md]:
```suggestion
I encourage all to embrace Sass modules! Here are a few links to get you started:
```

File: _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md (hunk 2 above)
Comment (hougasian) [reply _posts/2019-11-01-understanding-dart-sass-modules-and-name-spaced-variables.md]: jekyll wil l convert the twitter handle on it's own.
thanks for the other edits! :D

**Pull request merged**